chia/rpc/data_layer_rpc_api.py

```diff
@@ -310,3 +314,25 @@ async def get_kv_diff(self, request: Dict[str, Any]) -> EndpointResult:
         for rec in records:
             res.insert(0, {"type": rec.type.name, "key": rec.key.hex(), "value": rec.value.hex()})
         return {"diff": res}
+
+    async def add_mirror(self, request: Dict[str, Any]) -> EndpointResult:
+        store_id = request["id"]
+        id_bytes = bytes32.from_hexstr(store_id)
+        urls = request["urls"]
+        amount = request["amount"]
+        fee = get_fee(self.service.config, request)
+        await self.service.add_mirror(id_bytes, urls, amount, fee)
+        return {}
+
+    async def delete_mirror(self, request: Dict[str, Any]) -> EndpointResult:
+        coin_id = request["id"]
+        id_bytes = bytes32.from_hexstr(coin_id)
```

Comment:
Is this really the coin id?  Doesn't the coin id change with every update?

Reply:
It's the coin ID of the mirror.  Each new mirror creation creates a coin ID to use to delete it.  Does matter what the singleton does in the meantime.